.github/workflows/on_push_docs.yml

```diff
@@ -44,7 +44,7 @@ jobs:
     runs-on: ubuntu-latest-16-cores
     strategy:
       matrix:
-        toolchain: ["nightly-2025-02-05"]
+        toolchain: ["nightly-2025-09-16"]
```

Comment:
Could you add a comment to `rust-toolchain` to also bump this version when bumping to new stable?